server/lib/whispers/whisper-service.ts

```diff
@@ -43,6 +48,7 @@ export default class WhisperService {
   private sessionUsers = IMap<SbUserId, ISet<SbUserId>>()
 
   constructor(
+    private chatService: ChatService,
```

Comment:
This dependency doesn't seem to be used? And it would be incredibly weird to have this dependency anyway